lib/bundle/cask_dumper.rb

```diff
@@ -13,15 +13,33 @@ def casks
 
       require "cask/caskroom"
 
-      @casks ||= Cask::Caskroom.casks.map(&:to_s)
-      @casks.map { |cask| cask.chomp " (!)" }
-            .uniq
+      @casks ||= Cask::Caskroom.casks.map(&:to_s).uniq
+    end
+
+    def casks_config
+      return {} unless Bundle.cask_installed?
+
+      @casks_config ||= Hash[Cask::Caskroom.casks.map(&:to_s).zip(
+        Cask::Caskroom.casks.map do |cask|
+          if cask.config.nil? || cask.config.explicit.blank?
+            {}
+          else
+            cask.config.explicit.map do |k, v|
+              "#{k}: \"#{v.sub(/^#{ENV['HOME']}(#{File::SEPARATOR}|$)/, "~/")}\""
```

Comment:
Is `#{File::SEPARATOR}` just `/`? If so: I'd just use that.

Reply:
This I lifted from [cask/artifact/relocated.rb](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cask/artifact/relocated.rb#L99).

Reply:
I wonder if we could make that a public, documented method and call it from here?